frontend/www/js/omegaup/components/contest/NewForm.vue

```diff
@@ -485,6 +496,29 @@ export default class NewForm extends Vue {
     }
     this.$emit('create-contest', request);
   }
+
+  blockCatLanguage(): void {
+    for (const problem of this.initialProblems) {
+      if (problem.languages.split(',').includes('cat')) {
+        this.catLanguageBlocked = true;
+        break;
+      }
+    }
+  }
+
+  onRemove(language: string) {
+    if (
+      !this.catLanguageBlocked ||
+      (this.catLanguageBlocked && language !== 'cat')
+    ) {
```

Comment:
```suggestion
    if (this.catLanguageBlocked && language == 'cat') {
      this.$emit('language-remove-blocked', language);  // así se puede mostrar un warning de que eso no se permite.
      return;
    }
```

siempre que se pueda, se prefiere el uso del [guard clause pattern](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html)

Reply:
incluyo `language` en el `$emit` aunque siempre es `'cat'`?

Reply:
sí porfa. así se puede agregar a la cadena que muestra el error